features/notifications.yml

```
@@ -1,38 +1,52 @@
 name: Notifications
-description: The notifications API sends system notifications to the user, often even when the page or browser is not the foreground application.
-spec: https://notifications.spec.whatwg.org/
-caniuse: notifications
-# Notifications support is complicated and this entry is unusual. It represents
-# some kind of notification support available to web pages. More fine-grained
-# features could be added to represent specific things that work.
-#
-# https://caniuse.com/notifications marks iOS support as partial with this note:
-# "Requires website to first be added to the Home Screen."
+# This feature is the general Notifications API, and does not anchor to the
+# Caniuse feature.
 #
-# BCD doesn't use partial_implementation for safari_ios, but has this note:
-# "Notifications are supported in web apps saved to the home screen."
+# See `features/notifications-pwa.yml` for the Notifications feature for mobile
+# browsers that have platform-specific requirements and aligns with Caniuse.
+description: The Notifications API displays messages to the user outside of the browser, typically using the notification system of the host operating system. This feature is solely client-side, and does not integrate with server-originating notifications.
```

```suggestion
description: The Notifications API displays messages to the user, typically using the notification system of the host operating system.
```

Trying to make this shorter, in order to reuse it almost as is in the other feature. My reasoning is that it may be hard for users to understand why there are 2 related features. If we make the descriptions the same, but just add a bit about service worker/install in the other one, then that might help.